javascript/version_checker.js

```diff
@@ -0,0 +1,75 @@
+import Toastify from "toastify-js/src/toastify-es.js"
+import CableReady from "cable_ready"
+
+CableReady.operations.stimulusReflexVersionMismatch = operation => {
+  const levels = {
+    "info": {},
+    "success": { background: "rgb(0, 176, 155)" },
+    "warn": { background: "rgb(247, 187, 60)" },
+    "error": { background: "rgb(255, 95, 109)" },
```

Comment:
Can you make sure this color is accessible with the text? I use a bookmarklet called tota11y.

Reply:
Good call, will update it, thank you!

Reply:
I think this should be a lot better!
![Screenshot 2023-03-13 at 15 43 43](https://user-images.githubusercontent.com/6411752/224738432-f4568466-5d08-4735-8ce5-7ff01cc98d5a.png)
![Screenshot 2023-03-13 at 15 43 28](https://user-images.githubusercontent.com/6411752/224738458-a1e2f1eb-73a5-4b4f-b96b-f5ed64f6ed63.png)